lib/server.js

```diff
@@ -104,6 +88,12 @@ Server.prototype._assertHaveAllTasks = function () {
 Server.prototype._subscribe = function (queueName) {
   debug('_subscribe(' + queueName + ')')
   this.log.trace('ponos.Server: subscribing to ' + queueName)
+  if (!this._tasks[queueName]) {
+    this.log.warn({
+      queueName: queueName
+    }, 'ponos.Server: handler not defined')
+    return
+  }
```

Comment:
Discussed with @bkendall offline, this.hermes.subscribe is synchronous therefore the documentation here shouldn't indicate the function returns a promise